CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### 🚀 Added
+- Add test for two keys that only differ in case [#205] (https://github.com/dotenv-linter/dotenv-linter/pull/263) [@yanakad] (https://github.com/yanakad)
```

Comment:
```suggestion
- Add test for two keys that only differ in case [#269](https://github.com/dotenv-linter/dotenv-linter/pull/269) ([@yanakad] (https://github.com/yanakad))
```